tests/unit/schema/wrappers/test_method.py

```diff
@@ -123,6 +123,16 @@ def test_method_paged_result_field_no_page_field():
                          )
     assert method.paged_result_field is None
 
+    method = make_method('Foo',
```

Comment:
Formatting nit: please add a newline and then indent here for better readability

Reply:
Done.. I think.